File: TaskManager/src/app/project.ts
import { Timestamp } from "rxjs";

export class Project {
    projectID:number;
    projectName:string;
    dateOfStart:string;
    teamSize:number;

    constructor() {
        this.projectID = 0;
        this.projectName = '';
        this.dateOfStart = '';
        this.teamSize = 0;
    }
}

File: TaskManagerApp_Backend_code/src/main/java/com/taskmanager/App/Entities/Project.java
package com.taskmanager.App.Entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
public class Project {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int projectID;
	private String projectName;
	private String dateOfStart; 
	private int teamSize;
}
